src/plugins/console/public/application/containers/editor/monaco/monaco_editor_actions_provider.test.ts

```diff
@@ -147,4 +147,67 @@ describe('Editor actions provider', () => {
       expect(link).toBe(docsLink);
     });
   });
+
+  describe('provideCompletionItems', () => {
```

Comment:
Tests should be improved to not have to mock al of the monaco internal funcitons, I opened https://github.com/elastic/kibana/issues/182167 to track this.